package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bbc/psammead",
-  "version": "4.1.5",
+  "version": "4.1.6",
```

Comment:
I just merged in some changes so that the root package version bumping is not required